woocommerce/woocommerce-android
# Analytics button improvements

Closes: #7546


### Description

This PR improves the entry point button for Analytics Hub. It's a result of this conversation (internal): p1666012627363459-slack-C02KUCFCSFP

### Testing instructions

1. Run the app
2. Tap on "See more" while on the "Today" tab
3. **Assert that "today" analytics hub is opened**
4. Navigate back and then to any other tab, e.g. "this month"
5. Tap on "See more"
6. **Assert that Analytics Hub is opened with corresponding time-range selected**

### Images/gif


https://user-images.githubusercontent.com/5845095/197338109-dd24ebae-6872-4110-8a5b-b63a15c35615.mov
